### ISSUE TITLE
Activating TbSync is broking IMAP pull of the email account

### ISSUE THREAD

**Issue by Flaburgan** [jobisoft/TbSync#467]
## Your environment

TbSync version: 2.18
Thunderbird version: 78.4.2

[ ] Yes, I have installed the latest available beta version from 
https://tbsync.jobisoft.de
and my issue is not yet fixed, I can still reproduce it.

I installed TbSync and the provider for Exchange ActiveSync, it works well thank you.
However, since I did that, the linked email account (which is set on IMAP) isn't able to pull new e-mails anymore. Is that a known issue? How can I solve that situation?

**Comment by jobisoft** [comment]
That is not a known issue. I do not interact with IMAP in any way. What happens if you disable/remove TbSync again?

**Comment by Flaburgan** [comment]
Disabling TbSync and the provider doesn't solve the issue. Note that my other IMAP accounts have no problem, it's only the one which had TbSync activated where pulling e-mail isn't working anymore. Could it be possible that to add Exchange sync changed some settings on the server, or something like this?

**Comment by jobisoft** [comment]
Is there a direct relationship between installing TbSync and IMAP not working anymore? Can you check with your IT, if they disabled IMAP (Exchange servers can do that)?

**Comment by Flaburgan** [comment]
>Is there a direct relationship between installing TbSync and IMAP not working anymore?

I think so yeah. I started my new job last week, setup the email account with IMAP, it was working fine. Then I discovered the Exchange calendar, did a quick search and found your addons (which are working great btw). I installed them maybe one or two days after I set up IMAP, and since then, no new e-mails reached my inbox (but I can see new emails in the webmail).

**Comment by Flaburgan** [comment]
Wow, sorry, actually, I forgot I also had to upgrade Thunderbird to make TbSync works. This is the upgrade which broke the pulling, as it now requires TLS 1.2. See https://support.mozilla.org/en-US/kb/thunderbird-78-faq#w_after-upgrading-to-thunderbird-78-i-cannot-get-or-send-email-messages After following those steps, I am now able to pull emails again! Closing this, sorry for the loss of your time. Have a nice day.
